gateway/src/apicast/conditions/operation.lua

```diff
@@ -23,7 +23,13 @@ local evaluate_func = {
   -- false otherwise.
   ['matches'] = function(left, right)
     return (match(tostring(left), tostring(right)) and true) or false
-  end
+  end,
+
+  -- Numerics entries
```

Comment:
I'm not sure it's a good idea to convert to 0 any strings that cannot be converted into numbers.

I can see how this can become confusing in policies that expose these operations. Some users might assume that comparisons between strings work and that 'a' < 'b' should evaluate to true, for example.

Should we validate in each policy that these operations are always called with integers and discard the operation if it contains strings?

Reply:
I'm still concerned about this. I think at the very least we should add an explanation in the README.